ui/ducks/app/app.js

```diff
@@ -101,6 +102,24 @@ export default function reduceApp(state = {}, action) {
         qrCodeData: action.value,
       };
 
+    case actionConstants.SET_UNSIGNED_TRANSACTIONS_AND_ESTIMATES:
+      return {
+        ...appState,
+        unsignedTransactionsAndEstimates: action.payload,
+      };
+
+    case actionConstants.SET_SMART_TRANSACTIONS_LIVENESS:
+      return {
+        ...appState,
+        smartTransactionsLiveness: action.payload,
+      };
+
+    case actionConstants.SET_SMART_TRANSACTIONS_ERROR:
+      return {
+        ...appState,
+        smartTransactionsError: action.payload,
+      };
```

Comment:
Instead of adding STX state directly into the appState, can I use a namespace for Smart Transactions, e.g.:
```
case actionConstants.SET_UNSIGNED_TRANSACTIONS_AND_ESTIMATES:
  return {
    ...appState,
    smartTransactionsState: {
      ...appState.smartTransactionsState,
      unsignedTransactionsAndEstimates: action.payload,
    }
  };
```